### ISSUE TITLE
OTR/NTR: ciliary transition zone protein complexes

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#9312]
Hi, I would like to rename the GO:0036038 tectonic complex into 'tectonic-like complex'. A new paper published a similar complex which they named
B9 complex \(PubMed=22179047\) and characterizes its function

Many thanks

Sylvain

Here are the modifications I propose:

GO:0036038 tectonic-like complex
A protein complex that is located at the ciliary transition zone and consists of tectonic proteins \(TCTN1, TCTN2\), B9 domain-containing proteins \(B9D1, B9D2, MKS1\) and other proteins. Acts as a barrier that prevents diffusion of transmembrane proteins between the cilia and plasma membranes.

PubMed=22179047;

Synonym=B9 complex

Reported by: sylvainpoux

Original Ticket: [geneontology/ontology-requests/9103](https://sourceforge.net/p/geneontology/ontology-requests/9103)

**Comment by vanaukenk** [comment]
Hi,

I was annotating a paper on C. elegans ciliary transition zone proteins and would like to make some annotations to the appropriate protein complexes that reside and function there.

First, are we still adding new protein complex terms to the CC ontology?

If yes, then from some reading, it looks like there are two main protein complexes, referred to in recent papers as 'modules', that would be appropriate for annotation:  the MKS/JBTS module and the NPHP module.

I believe the MKS/JBTS module corresponds to GO:0036038 and so I'm wondering if we should re-name this term, since it looks like the MKS nomenclature is what is currently used in the literature. 
The NPHP module could be broken down into two separate modules, the NPHP1-4-8 module and the NPHP5-6 module.  Both would be new CC terms, also children of GO:0035869 ciliary transition zone.

If we make these changes, I don't know if we want to use the 'module' nomenclature or just refer to these as the MKS/JBTS complex and the NPHP complex.  I think the reason 'module' is used in the literature is that:

"...proteins in the MKS–JBTS module (green) mostly interact with other proteins within the same module and only rarely with those in the NPHP module (each line designates a reported protein–protein interaction)."  

In other words, the modules are not wholly distinct complexes, but are believed to have some association between them, albeit one that is considerably less strong than the association within the individual modules.  

See:   http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3373398
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3383065/

Please let me know if you'd like more information.

Thanks,
--Kimberly

**Comment by paolaroncaglia** [comment]
Hi Kimberly,

I have re-opened this ticket so it doesn't fall through the cracks, but I'm afraid I won't have a chance to look into it too soon. Please let me know if it is very urgent.

Thanks, and a good weekend,

Paola

**Comment by vanaukenk** [comment]
Thanks, Paola.

I can annotate the gene products to the parent term, 'ciliary transition zone', in the meantime, so this is not urgent.  

If you need any more info, let me know.

Enjoy your weekend, too.

--Kimberly

**Comment by paolaroncaglia** [comment]
@vanaukenk @krchristie @JohnvanDam 

Hi, please see Kimberly’s comment dated Nov 6, 2015.
Questions:

1) Are the MKS–JBTS and NPHP modules/protein complexes *stable* complexes, or are they transient? We tend to consider transient complexes outside the scope of GO. 
(See http://wiki.geneontology.org/index.php/Guidelines_on_%27protein_complex%27_terms)

2) If they are stable complexes, would they be useful for annotation? 

3) Is MKS–JBTS the same as GO:0036038 TCTN-B9D complex? If yes, is the former the preferred nomenclature in cilia literature?

4) If this applies: please suggest definitions for MKS–JBTS and/or NPHP modules/protein complexes, and functions that they are capable of if known, and processes they are involved in if known.

Thanks,
Paola

**Comment by krchristie** [comment]
Hi Paola,

I'm under the impression that the MKS and NPHP modules are stable complexes involved in the structure of the transition zone that separates the ciliary plasm from the cytoplasm. Here is an interesting paper discussing them.

-Karen

Barker AR, Renzaglia KS, Fry K, Dawe HR. Bioinformatic analysis of ciliary
transition zone proteins reveals insights into the evolution of ciliopathy
networks. BMC Genomics. 2014 Jun 26;15:531. doi: 10.1186/1471-2164-15-531. PubMed
PMID: 24969356; PubMed Central PMCID: PMC4092220.

**Comment by paolaroncaglia** [comment]
Thanks Karen. The reference you suggest 
http://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-531
is very informative, but I feel that we still don’t have enough elements to define these complexes in GO, and I’m not sure it’d be useful to have these terms in until more is known about their function (we don't have facts to link them to precise processes either). These uncertainties are summarised in the conclusions:

“Given the near-ubiquity of TMEM67, CC2D2A, B9D1, B9D2, AHI1, a single TCTN family member, and possibly MKS1 across ciliated eukaryotes [see fig 2], we suggest a dynamic complex built around these proteins and varying in a spatio-temporal and/or organism/tissue-specific manner. In concordance with current ideas, formation of this complex is likely to be key in the proper formation and functioning of ciliary Y-links.”

Therefore, @vanaukenk, I’d rather not add ‘MKS/JBTS module’ and ‘NPHP module’ to GO, but I could add them (and ‘MKS module') as narrow or related synonyms of ‘ciliary transition zone’, adding the papers you and Karen quoted as dbxref. Let me know if you think this would be helpful.

Thanks,

Paola

**Comment by vanaukenk** [comment]
Hi @paolaroncaglia,
There are a couple of new papers that have come out this year about the role of these different modules in transition zone assembly, so if it's alright, I'd like to just take a look at those papers to come back up to speed on this issue.
In the literature, people tend to talk about the MKS and NPHP complexes as separate functional units based on several criteria, e.g. genetic interactions, physical interactions, and localization dependencies, but I take your point that we want good definitions of these modules, from both a compositional and functional point of view, to consider adding them to GO.
Thanks,
--Kimberly

**Comment by JohnvanDam** [comment]
1) Are the MKS–JBTS and NPHP modules/protein complexes stable complexes, or are they transient? We tend to consider transient complexes outside the scope of GO. 
(See http://wiki.geneontology.org/index.php/Guidelines_on_%27protein_complex%27_terms)

They are considered stable complexes, or at least stably co-localise to the transition zone and influence each others localisation. I have recently been involved in a large study to identify a novel component of the MKS module and the very nice thing was that my collaborator did some very systematic experiments. The NPHP and MKS modules are very hierarchical in that some components can be removed while module integrity is maintained causing only minor phentypes, while others will disrupt the whole system and cause severe phenotypes including prenatal death. We are only now finding out how these proteins are related to actual structures (before they were only found to be "required" for Y-shaped linker formation, but we now finally find evidence that they are actual "part of" that structure, this is still thin though). Literature wise, these modules are becoming increasingly important in ciliary biology because of their strong relation to human genetic disease.

2) If they are stable complexes, would they be useful for annotation?
They are useful for annotation in that there is a lot of evidence that suggest that these are functional units and are often found mutated in human genetic disease (ciliopathies). They perform functions in ciliary protein transport, protein barrier function, and cellular signalling (although that could likely be indirect)

3) Is MKS–JBTS the same as GO:0036038 TCTN-B9D complex? If yes, is the former the preferred nomenclature in cilia literature?

The TCTN-BD9 complex seems to be (according to genes assigned) a more "broad" definition of the MKS module as how we used it. The NPHP module is separate and also not part of the TCTN-B9D. I think "MKS module" or "MKS complex" is more common in literature, but have no numbers.

4) If this applies: please suggest definitions for MKS–JBTS and/or NPHP modules/protein complexes, and functions that they are capable of if known, and processes they are involved in if known.

I will get back to you this afternoon with specific proposal for definitions of these complexes including references, parent and child terms etc.

In the meantime this is the paper I have been involved in: 
Lambacher NJ, Bruel A-L, van Dam TJP, Szymańska K, Slaats GG, Kuhns S, et al. TMEM107 recruits ciliopathy proteins to subdomains of the ciliary transition zone and causes Joubert syndrome. Nat. Cell Biol. Nature Publishing Group; 2015;18:122–31. http://dx.doi.org/10.1038/ncb3273

**Comment by JohnvanDam** [comment]
So many distractions... I'll get you the proposed definitions tomorrow. Sorry for this.

**Comment by JohnvanDam** [comment]
"TCTN-B9D" -> no hits in pubmed
"MKS-complex" -> 4 hits
"mks-module" -> no hits
"mks-jbts" -> no hits
"mks module" -> 7 hits

“nphp module” -> 6 hits
“nphp-module” -> 0 hits
“nphp-complex” -> 0 hits
“nphp complex” -> 21 hits

The TCTN-B9D complex comprises many elements of the so called MKS module, so we can consider these to be essentially the same, which some exceptions. The distinction between NPHP module and MKS module comes from the fact that the proteins of either module can be deleted/knocked down without loss of structure, however when both modules are disrupted (e.g. by double gene knockout like NPHP4 and MKS1) the transition zone becomes disorganised and y-shaped linkers disappear or are misaligned or dislodged from membrane and axoneme (genetic interaction). Between most module components physical interactions have been measured, but the question remains if this is transient in nature. The localisation of many MKS/NPHP module components are highly restricted to the TZ, but have only recently been shown to be “fixed” within the structure for one or two components, suggesting to some extent a non-transient nature of the modules. It has been shown that a new component of the MKS module (TMEM107) is fixed at the y-shaped attachments to the membrane suggesting that the MKS module may form the anchor of the Y-shape to the membrane. Since many MKS proteins have transmembrane regions this may very well be. The NPHP module is smaller in size and to my knowledge none of the components contain TM regions.

Currently MKS module seems to be used more frequently, but I think this would be counter the normal nomenclature in GO (i.e. complex). I suggest to drop the TCTN-B9D name since it is not used in literature. CEP290 is currently annotated as being part of the TCTN-B9D complex. The CEP290 protein is generally not considered “part of” the MKS module although it is tightly associated to it functionally (PMID: 26982032).  Some other TZ components that are not considered to be part of this complex are also assigned to this term like AHI1.

Overall I think the TCTN-B9D term needs an overhaul. Below is my suggested name and definition including new references.

Name: MKS module
Ontology: cellular_component
synonyms: MKS complex, tectonic complex, tectonic-like complex
Definition: A protein complex that is located at the ciliary transition zone and consists of several proteins some of which are membrane bound. Acts as an organiser of transition zone inner structure, specifically the Y-shaped linkers, in conjunction with the NPHP complex. The complex also acts as part of the selective barrier that prevents diffusion of proteins between the ciliary cytoplasm and cellular cytoplasm as well as between the ciliary membrane and plasma membrane.
Source (old): PMID:22179047, PMID:21725307
Source (new): PMID:26595381, PMID:21422230, PMID:25869670, PMID:26982032, PMID:21565611
Comment: Although there is some evidence, it is unclear whether the MKS- and NPHP complexes are truly “part of” the y-shaped linkers or are just responsible for aligning and attaching the y-shaped linkers to the membrane and axoneme.
relationships:
is_a protein complex
part_of ciliary transition zone
is_a ciliary part

I'll draft a suggested definition of the NPHP module next. I suggest to not define the "newer" CEP290 module in GO right now, since this term is, I believe, not widespread.

Apologies for the long read. I'll try to keep it shorter next time.

**Comment by JohnvanDam** [comment]
Found this while searching for references. I am adding it to this thread as a side note:
Williams CL, Li C, Kida K, Inglis PN, Mohan S, Semenec L, et al. MKS and NPHP modules cooperate to establish basal body/transition zone membrane associations and ciliary gate function during ciliogenesis. J. Cell Biol. 2011;192:1023–41.

**Comment by paolaroncaglia** [comment]
Hi @JohnvanDam ,

I edited the existing term GO:0036038 TCTN-B9D complex based on your input. The new stanza is below. (Please note, we’d rather have ‘complex’ in the primary name for consistency with other protein complex terms; I made the ‘module’ wording an exact synonym.)
When you have a chance, could you please suggest definition etc. for the NPHP module/complex, as you kindly offered to? :-)
Thanks.

[Term]
id: GO:0036038
name: MKS complex
namespace: cellular_component
def: "A protein complex that is located at the ciliary transition zone and consists of several proteins some of which are membrane bound. Acts as an organiser of transition zone inner structure, specifically the Y-shaped links, in conjunction with the NPHP complex. The MKS complex also acts as part of the selective barrier that prevents diffusion of proteins between the ciliary cytoplasm and cellular cytoplasm as well as between the ciliary membrane and plasma membrane." [GOC:cilia, GOC:sp, PMID:21725307, PMID:22179047, PMID:26595381, PMID:21422230, PMID:25869670, PMID:26982032, PMID:21565611]
comment: Although there is some evidence, it’s still unclear if the MKS and NPHP complexes are constituents parts of the Y-shaped links or are simply responsible for aligning and attaching the Y-shaped links to the membrane and axoneme.
synonym: "B9 complex" NARROW []
synonym: "MKS module" EXACT []
synonym: "TCTN-B9D complex" BROAD []
synonym: "tectonic complex" NARROW []
synonym: "tectonic-like complex" NARROW []
is_a: GO:0043234 ! protein complex
is_a: GO:0044441 {is_inferred="true"} ! ciliary part
relationship: part_of GO:0035869 ! ciliary transition zone

**Comment by paolaroncaglia** [comment]
Note for self: 
Once the NPHP term is added, I can close this ticket.

**Comment by JohnvanDam** [comment]
Name: NPHP complex
namespace: cellular_component
def: "A protein complex that is located at the ciliary transition zone and consists of the NPHP4 and NPHP1 proteins. Acts as an organiser of the transition zone inner structure, specifically the Y-shaped links, in conjunction with the MKS complex. Mutation or deletion of individual NPHP complex components cause disruption in ciliary protein trafficking, but cause large structural defects when MKS complex components are also mutated. The NPHP complex is required for correct functioning of the WNT and Hippo signaling pathways." [PMID:21422230,PMID:25150219,PMID:21555462,PMID:21498478,PMID:18337471]
comment: Although there is some evidence, it’s still unclear if the MKS and NPHP complexes are constituents parts of the Y-shaped links or are simply responsible for aligning and attaching the Y-shaped links to the membrane and axoneme.
synonym: "NPHP module"
is_a: GO:0043234 ! protein complex
is_a: GO:0044441 {is_inferred="true"} ! ciliary part
relationship: part_of GO:0035869 ! ciliary transition zone

**Comment by paolaroncaglia** [comment]
Thanks @JohnvanDam ! 
I added GO:1990957 ‘NPHP complex’ as below (I modified your suggested definition slightly so we don’t quote mutagenesis experiments directly, but we say what the results of those experiments indicate; and I added links to the relevant biological processes). Closing now. Here’s the full stanza for reference:

[Term]
id: GO:1990957
name: NPHP complex
namespace: cellular_component
def: "A protein complex that is located at the ciliary transition zone and consists of the NPHP4 and NPHP1 proteins. It acts as an organiser of the transition zone inner structure, specifically the Y-shaped links, in conjunction with the MKS complex. It is involved in ciliary protein trafficking and is required for correct functioning of the WNT and Hippo signaling pathways." [GOC:cilia, PMID:18337471, PMID:21422230, PMID:21498478, PMID:21555462, PMID:25150219]
comment: Although there is some evidence, it's still unclear if the MKS and NPHP complexes are constituents parts of the ciliary Y-shaped links or are simply responsible for aligning and attaching the Y-shaped links to the cilium membrane and axoneme.
synonym: "NPHP module" EXACT []
is_a: GO:0043234 ! protein complex
is_a: GO:0044441 {is_inferred="true"} ! ciliary part
relationship: capable_of_part_of GO:0016055 ! Wnt signaling pathway
relationship: capable_of_part_of GO:0035329 ! hippo signaling
relationship: capable_of_part_of GO:0061512 ! protein localization to cilium
relationship: part_of GO:0035869 ! ciliary transition zone
